modules/gutenberg/module.php

```diff
@@ -169,6 +154,19 @@ public function dequeue_assets() {
 		wp_dequeue_style( 'wc-blocks-style' );
 	}
 
+	/**
+	 * Check whether the "Optimized Gutenberg Loading" settings is enabled.
+	 *
+	 * The 'elementor_optimized_gutenberg_loading' option can be enabled/disabled from the Elementor settings.
+	 * For BC, when the option has not been saved in the database, the default '1' value is returned.
```

Comment:
@rami-elementor Why don't you check on which state the Experiment was on to decide the default setting?